services/wfm/src/main/java/org/openkilda/wfm/topology/event/OFELinkBolt.java

```diff
@@ -578,6 +576,11 @@ private void handleMovedIsl(Tuple tuple, String srcSwitch, int srcPort, String d
         passToTopologyEngine(tuple, message);
     }
 
+    private void handleSentDiscoPacket(DiscoPacketSendingConfirmation confirmation) {
+        logger.debug("Discovery packet is sent from {}", confirmation);
```

Comment:
I would like to alter message to something like "Got confirmation - discovery packet from {} have been sent"